props/saber_fett263_buttons.h

```diff
@@ -104,20 +248,53 @@ SaberFett263Buttons() : PropBase() {}
         SaberBase::SetLockup(SaberBase::LOCKUP_NONE);
         auto_melt_on_ = false;
       }
+
+      // EVENT_PUSH
+      if (fabs(mss.x) < 3.0 && 
+          mss.y * mss.y + mss.z * mss.z > 120 &&
+          fusor.gyro().len2() < 1000 &&
+          fusor.swing_speed() < 30 &&
+          fusor.gyro().x < 10) {    
+        Event(BUTTON_NONE, EVENT_PUSH);
+      }
+
     } else {
-      // Swing On gesture control this portion allows fine tuning of speed needed to ignite
+      // EVENT_SWING - Swing On gesture control to allow fine tuning of speed needed to ignite
       if (millis() - saber_off_time_ < MOTION_TIMEOUT) {
         SaberBase::RequestMotion();
-        // Edit '250' value in line below to change swing on sensitivity when off
-        // 250 ~ 400 work best in testing
-        if (!swinging_ && fusor.swing_speed() > 250) {
+        if (!swinging_ && fusor.swing_speed() > FETT263_SWING_ON_SPEED) {
           swinging_ = true;
           Event(BUTTON_NONE, EVENT_SWING);
         }
       }
+      // EVENT_THRUST
+      if (mss.y * mss.y + mss.z * mss.z < 16.0 &&
+        mss.x > 14  &&
+        fusor.swing_speed() < 150) {
+        Event(BUTTON_NONE, EVENT_THRUST);
+      }
     }
   }
 
+// Fast On Gesture Ignition
+  virtual void FastOn() {
+    if (IsOn()) return;
+    if (current_style() && current_style()->NoOnOff())
+      return;
+    activated_ = millis();
+    STDOUT.println("Ignition.");
+    MountSDCard();
+    EnableAmplifier();
+    SaberBase::RequestMotion();
+    // Avoid clashes a little bit while turning on.
+    // It might be a "clicky" power button...
+    IgnoreClash(500);
+    SaberBase::TurnOn();
+    // Optional effects
+    SaberBase::DoEffect(EFFECT_FAST_ON, 0);
+    hybrid_font.PlayCommon(&SFX_faston);
```

Comment:
This should probably be in SB_Effect()

Reply:
I thought you replaced SB_Effect()?

Reply:
You have an SB_Effect() function further down in this file....

Reply:
I see I was thinking the old syntax